spec/features/restaurants/registered_user_views_a_restaurant_page_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
+
+describe 'As a registered user' do
+
+  before(:each) do
+    VCR.use_cassette('geocode_lookup', record: :all) do
+      @user = create(:user)
```

Comment:
Just a tip - you can use (outside of the before(:each)) let(:user) { create(:user) } which 'lazy loads' the user once, and doesn't allow changes (look up let! to allow changes). When doing it this way, it's only created once, instead of before each test, which improves test speed.